src/options.cpp

```diff
@@ -138,6 +142,46 @@ void global_settings_via_options::set_oauth_10_support(const po::variables_map &
   }
 }
 
+void global_settings_via_options::set_bytes_per_sec(const po::variables_map &options) {
```

Comment:
Seems like I commented on this in your fork, which is why it doesn't show up in your pull request.

You need to call both new setter methods from global_settings_via_options::set_new_options, line 43 in the same file, otherwise we're not updating our option values with the externally provided values.

Reply:
Didn't have time to test changes, will fix & test it tonight